xarray/tests/indexes.py

```diff
@@ -0,0 +1,79 @@
+from collections.abc import Hashable, Iterable, Sequence
```

Comment:
refactored these out from `test_dataset.py` so they can be reused

Reply:
Nice!